tests/integration/logging/test_zmq_logging.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+
+# Import Python libs
+from __future__ import absolute_import, print_function, unicode_literals
+
+import logging
+import multiprocessing
+import threading
+import time
+
+import zmq
+import msgpack
+
+import salt.log.setup
+import salt.log.handlers
+
+from tests.integration import get_unused_localhost_port
+from tests.support.unit import TestCase
+
+
+class TestLogHandler(logging.NullHandler):
+
+    def __init__(self, level=logging.NOTSET):
+        super(TestLogHandler, self).__init__(level=level)
+        self.messages = []
+
+    def handle(self, record):
+        self.messages.append(record)
+
+
+def test_target(host, port):
```

Comment:
If this isn't a test don't name it like `test_*`, once we get this into neon and under pytest we'd have to rename it.